### ISSUE TITLE
'npm run watch' not working

### ISSUE THREAD

**Issue by ameen06** [jeffreyvr/tailpress#128]
Showing the error:

> tailpress@3.0.0 watch
> cross-env NODE_ENV=development concurrently "npm run watch:css-app" "npm run watch:css-editor" "npm run watch:js"

[0] 
[0] > tailpress@3.0.0 watch:css-app
[0] > cross-env NODE_ENV=development tailwindcss -i ./resources/css/app.css -o ./css/app.css --postcss --watch
[0]
[2]
[2] > tailpress@3.0.0 watch:js
[2] > cross-env NODE_ENV=development ./node_modules/.bin/esbuild ./resources/js/app.js --bundle --outfile=./js/app.js --watch
[2]
[1]
[1] > tailpress@3.0.0 watch:css-editor
[1] > cross-env NODE_ENV=development tailwindcss -i ./resources/css/editor-style.css -o ./css/editor-style.css --postcss --watch
[1]
[0] The filename, directory name, or volume label syntax is incorrect.
[2] The filename, directory name, or volume label syntax is incorrect.
[1] The filename, directory name, or volume label syntax is incorrect.
[0] npm run watch:css-app exited with code 1
[1] npm run watch:css-editor exited with code 1
[2] npm run watch:js exited with code 1

**Comment by jeffreyvr** [comment]
I'm not able to reproduce this issue. Can you provide more information, like the OS are you using and which node version?

**Comment by justingolden21** [comment]
I'm on Windows and get this output for `npm run watch`:

```
C:\Users\justin.golden\Local Sites\hello-world\app\public\wp-content\themes\example-theme>npm run watch

> tailpress@0.1.0 watch
> cross-env NODE_ENV=development concurrently "npm run watch:css-app" "npm run watch:css-editor" "npm run watch:js"

[0] Error occurred when executing command: npm run watch:css-app
[0] Error: spawn cmd.exe ENOENT
[0]     at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
[0]     at onErrorNT (node:internal/child_process:477:16)
[0]     at processTicksAndRejections (node:internal/process/task_queues:83:21)
[1] Error occurred when executing command: npm run watch:css-editor
[1] Error: spawn cmd.exe ENOENT
[1]     at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
[1]     at onErrorNT (node:internal/child_process:477:16)
[1]     at processTicksAndRejections (node:internal/process/task_queues:83:21)
[2] Error occurred when executing command: npm run watch:js
[2] Error: spawn cmd.exe ENOENT
[2]     at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
[2]     at onErrorNT (node:internal/child_process:477:16)
[2]     at processTicksAndRejections (node:internal/process/task_queues:83:21)
[2] npm run watch:js exited with code -4058
[1] npm run watch:css-editor exited with code -4058
[0] npm run watch:css-app exited with code -4058

C:\Users\justin.golden\Local Sites\hello-world\app\public\wp-content\themes\example-theme>
```

**Comment by justingolden21** [comment]
I solved it by killing my Local WP sever instance, deleting the theme folder from my WP site, copying the theme (without `node_modules`) from where I installed to the WP site, reinstalling (`npm i`) and opening the server back up on Local WP.

As a test: When I replaced `pb-12` with `pb-10` the padding changed to 10 units, even though the `pb-1` utility class was not used elsewhere before, so I know it's generating the classes. Still have to refresh the page for it to appear but that's on Local WP. :)
